docs/reference/operator_parameters.md

```diff
@@ -110,8 +110,9 @@ Those are top-level keys, containing both leaf keys and groups.
 
 * **min_instances**
   operator will run at least the number of instances for any given Postgres
-  cluster equal to the value of this parameter. When `-1` is specified, no
-  limits are applied. The default is `-1`.
+  cluster equal to the value of this parameter, except for standby clusters
+  which run with one pod if `numberOfInstances` is set to 1. When `-1` is
```

Comment:
this exception in the context of this doc looks somewhat arbitrary, the credentials' limitation is  worth mentioning here